gammapy/data/ivoa.py

```diff
@@ -284,7 +314,11 @@ def _obscore_row(base_dir, single_obsID, obs_publisher_did, access_url, table):
 
 
 def to_obscore_table(
-    base_dir, selected_obs=None, obs_publisher_did=None, access_url=None
+    base_dir,
+    selected_obs=None,
```

Comment:
All parameters should be described in the docstring

Reply:
Done!